packages/docs/docs/entities/components-repo.md

```diff
@@ -210,12 +254,12 @@ myRepo.register(
 //...
 myRepo.remove('string-component');
 ```
+## replace
+Replace a previously registered component by component name.
 
-### replace
-
-You can replace component record with another one:
-
-```js
+replace ensures that the new component will have the same index (order) as the old one. It is important because by default <AutoView /> picks the last registered component in ComponentsRepo.
```

Comment:
```suggestion
`replace` ensures that the new component will have the same index (order) as the old one. It is important because by default `<AutoView />` picks the last registered component in `ComponentsRepo`.
```